scripts/splitControl.js

```diff
@@ -0,0 +1,132 @@
+/*global $ */
```

Comment:
This splitter is something that I want to start building with in multiple places in the site. I would prefer we take this file out into its own MR so we can iterate on it and get it in as a widget we can use with this MR and others later.

To that end, it would be good to get some simple, manual tests in `SETUP/tests/manual_web` we can use to test it, both as part of its development and later as we make changes to it. I'm envisioning at least three test files:
1. Two pane vertical split
2. Two pane horizontal split
3. Two pane vertical split with the right pane also split horizontally.

Reply:
Thanks Casey. I'll do that.